lti_consumer/models.py

```diff
@@ -232,6 +234,22 @@ class LtiConfiguration(models.Model):
         help_text='Enable LTI Proctoring Services',
     )
 
+    @cached_property
+    def ccx_master_configuration(self):
+        """
+        CCX master block LTI configuration.
+
+        On configurations with a CCX location, return the configuration of the original master course location.
+        if configurations is not from a CCX location or the configuration is not found, return None.
+        """
+        if not is_ccx_location(self.location):
+            return None
+
+        try:
+            return LtiConfiguration.objects.get(location=self.location.to_block_locator())  # pylint: disable=no-member
+        except LtiConfiguration.DoesNotExist:
+            return None
```

Comment:
This way you could get rid of the try-except. @kuipumu 
```suggestion
        return LtiConfiguration.objects.first(location=self.location.to_block_locator())  # pylint: disable=no-member
```

Reply:
@Jacatove The "first" method doesn't receive any argument, this fails with a TypeError exception